camelot/view/controls/editors/floateditor.py

```diff
@@ -214,7 +214,7 @@ def set_maximum(self, maximum):
     def set_focus_policy(self, focus_policy):
         if focus_policy is not None:
             spinBox = self.findChild(CustomDoubleSpinBox, 'spinbox')
-            spinBox.setFocusPolicy(focus_policy)
+            spinBox.setFocusPolicy(Qt.FocusPolicy(focus_policy))
```

Comment:
mss moeten we ons hier wel afvragen of de item model er nt moet voor zorgen dat er hier een Qt object binnen komt ... mss moeten we die enumerations ook als names in de dict steken ... (ik bedoel dan in de get_standard_item calls in de delegates)